statistics/sample.go

```diff
@@ -47,6 +48,10 @@ type SampleItem struct {
 	Handle kv.Handle
 }
 
+// EmptySampleItemSize is the size of empty SampleItem, please update it when change the data structure.
+// 96 = 72 (datum) + 8 (int) + 16
```

Comment:
we can remove this comment now.

Reply:
Just for my convenience to calc size and gap sometimes...

Reply:
ah yes, I can remove "please update...."

Reply:
Done